app/controllers/portal/learners_controller.rb

```diff
@@ -133,40 +141,18 @@ def multiple_choice_report
   
 
   def report
-    # PUNDIT_REVIEW_AUTHORIZE
-    # PUNDIT_CHOOSE_AUTHORIZE
-    # no authorization needed ...
-    # authorize Portal::Learner
-    # authorize @learner
-    # authorize Portal::Learner, :new_or_create?
-    # authorize @learner, :update_edit_or_destroy?
-    @portal_learner = Portal::Learner.find(params[:id])
-    @activity_report_id = nil
-    offering = @portal_learner.offering
-    unless offering.report_embeddable_filter.nil? || offering.report_embeddable_filter.embeddables.nil?
-      @report_embeddable_filter = offering.report_embeddable_filter.embeddables
-    end
-    unless params[:activity_id].nil?
-      activity = ::Activity.find(params[:activity_id].to_i)
-      @activity_report_id = params[:activity_id].to_i
-      unless activity.nil?
-        activity_embeddables = activity.page_elements.map{|pe|pe.embeddable}
-        if offering.report_embeddable_filter.ignore
-          @portal_learner.offering.report_embeddable_filter.embeddables = activity_embeddables
-        else
-          filtered_embeddables = offering.report_embeddable_filter.embeddables & activity_embeddables
-          filtered_embeddables = (filtered_embeddables.length == 0)? activity_embeddables : filtered_embeddables
-          @portal_learner.offering.report_embeddable_filter.embeddables = filtered_embeddables 
-        end
-        @portal_learner.offering.report_embeddable_filter.ignore = false
-      end
-    end
-    
-    respond_to do |format|
-      format.html # report.html.haml
-        reportUtil = Report::Util.reload(@portal_learner.offering)  # force a reload of this offering
-        @page_elements = reportUtil.page_elements
-    end
+    # This report is for the teacher at the moment so for authentication
+    # we just check pundit for offering report method. See reports_controller.rb
```

Comment:
Is this comment still relevant, or does this code now handle the student view of the report too?